IPython/utils/wildcard.py

```diff
@@ -20,9 +20,11 @@
 from IPython.utils.dir2 import dir2
 from .py3compat import iteritems
 
-def create_typestr2type_dicts(dont_include_in_type2typestr=["lambda"]):
+def create_typestr2type_dicts(dont_include_in_type2typestr=None):
```

Comment:
Again, this could be made a tuple (or a frozenset, if we wanted to get clever, as it's used for membership checks - but it doesn't matter for one item).

However, since this function is only called once, I'd be inclined just to hardcode the value.